src/config/config_service.py

```diff
@@ -139,7 +139,12 @@ def load_config_model(self, name, user, parameter_values=None, skip_invalid_para
 
     def _visit_script_configs(self, visitor):
         configs_dir = self._script_configs_folder
-        files = os.listdir(configs_dir)
+
+        files=[]
+        # Read config file from within directories too
+        for _root, _dirs, _files in os.walk(configs_dir, topdown=True):
+            for name in _files:
+                files.append( re.sub ( r'^.*/runners/', '', os.path.join( _root, name) ) )
```

Comment:
I think you can just append `name` here. It doesn't include parent path

Reply:
no, simply adding the name returns different results and causes the tests to fail

Reply:
Alright, true. I just wanted to avoid having forward slashes for windows compatibility.
In this case I would suggest to `files.append(os.path.join(_root, name))` 

And later, in the line 218, don't do `os.path.join(configs_dir, config_path)` and use config_path directly

PS could you change the formatting rules please, and remove whitespaces in parentheses? E.g. `( a, b, c )` => `(a, b, c)` It would be better to have the same style across the project, if possible